auth0/resource_auth0_guardian.go

```diff
@@ -347,34 +344,18 @@ func readGuardian(ctx context.Context, d *schema.ResourceData, m interface{}) di
 func updateGuardian(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*management.Management)
 
-	if err := updatePolicy(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-
-	if err := updateEmailFactor(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-	if err := updateOTPFactor(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-
-	if err := updatePhoneFactor(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-
-	if err := updateWebAuthnRoaming(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-
-	if err := updateWebAuthnPlatform(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-
-	if err := updateDUO(d, api); err != nil {
-		return diag.FromErr(err)
-	}
-
-	if err := updatePush(d, api); err != nil {
+	result := multierror.Append(
+		updatePolicy(d, api),
+		updateEmailFactor(d, api),
+		updateOTPFactor(d, api),
+		updateRecoveryCodeFactor(d, api),
+		updatePhoneFactor(d, api),
+		updateWebAuthnRoaming(d, api),
+		updateWebAuthnPlatform(d, api),
+		updateDUO(d, api),
+		updatePush(d, api),
+	)
```

Comment:
It's better if we try al of these updates and report on any accumulated errors instead of failing at the first and not doing any of the others.